jina/serve/executors/__init__.py

```diff
@@ -63,7 +60,11 @@ def register_class(cls):
                     f'{cls.__init__} does not follow the full signature of `Executor.__init__`, '
                     f'please add `**kwargs` to your __init__ function'
                 )
-            wrap_func(cls, ['__init__'], store_init_kwargs)
+            taboo = {'self', 'args', 'kwargs', 'metas', 'requests', 'runtime_args'}
```

Comment:
can we move this dict to outside this static method ? Just having a const dict somwhere else

Reply:
Actually I think it might not be convenient. We have 3 options:
* extract this code outside class
```python
taboo = {'self', 'args', 'kwargs', 'metas', 'requests', 'runtime_args'}
_defaults = get_default_metas()
taboo.update(_defaults.keys())
```
(I dislike it when users try to import something but something else happens underthehood
* initialize TABOO but update it each time and I also dislike this.
* keep as it is
I think it's mostly subjective choice

Reply:
anyways I already handled this issue in https://github.com/jina-ai/jina/pull/5189/commits/f140966149db11ebb9997f0a3532a5a25a4ae529